src/05-longitudinal-and-multi-site-studies.md

```diff
@@ -1,4 +1,4 @@
-# Longitudinal and multi-site studies
+# Longitudinal studies and multi-site studies
```

Comment:
Why this change? It's `Longitudinal and multi-site studies` everywhere else, see: https://github.com/bids-standard/bids-specification/blob/ece9e8737d15a4ca9e5453dc8c473abbd3e4dd91/mkdocs.yml#L34

I think this change should be reverted